Model/Contacto.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * This file is part of webportal plugin for FacturaScripts.
+ * Copyright (C) 2018 Carlos Garcia Gomez <carlos@facturascripts.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace FacturaScripts\Plugins\webportal\Model;
+
+/**
+ * A contact of FacturaScripts is a loggeable user.
+ *
+ * @author Francesc Pineda Segarra <francesc.pineda@x-netdigital.com>
+ */
+class Contacto extends \FacturaScripts\Core\Model\Contacto
+{
+    /**
+     * Password hashed with password_hash()
+     *
+     * @var string
+     */
+    public $password;
+
+    /**
+     * New password.
+     *
+     * @var string
+     */
+    public $newPassword;
+
+    /**
+     * Repeated new password.
+     *
+     * @var string
+     */
+    public $newPassword2;
+
+    /**
+     * Returns an user by email.
+     *
+     * @param string $email
+     *
+     * @return self|null
+     */
+    public function getByEmail(string $email): ?self
```

Comment:
Unnecesary. You can use loadFromCode('', [new databasewhere...])
And inconsistent with get() method, that return false when fails.